unstructured/documents/xml.py

```diff
@@ -65,8 +69,19 @@ def _read_xml(self, content):
         if self.document_tree is None:
             try:
                 document_tree = etree.fromstring(content, self.parser)
+                if "<pre>" and "</pre>" in content:
```

Comment:
Can we move the conditional logic for `<pre>` outside of the `try`/`except`? Mainly worried about masking legitimate exceptions if they occur within this block.

The `try`/`except` was a hacky fix to a double encoding issue a while back, wouldn't mind refactoring that at some point if we every get time to.

Reply:
Sounds reasonable to me!